docs/user_guide/HOWTO/Using_the_SIMP_Platform_RPM.rst

```diff
@@ -0,0 +1,133 @@
+.. _howto-use-the-simp-platform-rpm:
+
+===========================
+SIMP Platform RPM
+===========================
+
+Download Details
+---------------------------------
+
+Location
+********
+
+The SIMP Platform RPM can be downloaded `HERE
+<simp-platform-1.0.0-0.noarch.rpm>`_.
```

Comment:
This link won't function properly and the package cannot be found in the `yum/simp` space on the download server

Reply:
This link still won't function properly.

Reply:
* The hyperlink doesn't work because it is a relative link.  
* Just spell out  the actual URI here;  "HERE" is unhelpful as hyperlink text, and the URI itself is the true object of the sentence.
* I thought the SIMP Community platform RPM URL was https://download.simp-project.com/simp-release-community.rpm now.

```suggestion
https://download.simp-project.com/simp-release-community.rpm
```